migrations/sql/V2020.04.02_1216__FixAdminPermissions.sql

```diff
@@ -0,0 +1,3 @@
+INSERT INTO "orderlyweb_user_group_global_permission"
+SELECT ID FROM "orderlyweb_user_group_permission"
+WHERE user_group = 'Admin' and permission = 'documents.manage' or  permission = 'tags.manage'
```

Comment:
The last 2 migrations to add `tags.manage` and `documents.manage` added permissions wrongly - they added an entry into `orderlyweb_user_group_permission` but not into `orderlyweb_user_group_global_permission`, which fails to grant the permission and also breaks the add permission interface. This fixes that issue.

I think this mistake reinforces my feeling that this schema is pretty confusing and makes the code more complicated than it needs to be. I'd be in favour of refactoring so we just have the 3 tables corresponding to global, report, version permissions and doing away with the abstract `orderlyweb_user_group_permission` table. Thoughts?

Reply:
Fair enough!